Remove unnecessary comments on issue templates (#4557)
ppy/osu: .github/ISSUE_TEMPLATE/feature-request-issues.md

name: Feature Request

> about: Let us know what you would like to see in the game!

Reviewer: ```suggestion
about: Features you would like to see in the game!
```
